profiles.md

```diff
@@ -0,0 +1,208 @@
+# Researcher profiles Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Single Profile
+**GET /api/cris/profiles/<:eperson-uuid>**
+
+Provide information about a specific profile. Only the Administrators or the owner eperson can access this endpoint.
+
+The JSON response document is as follow
+
+```json
+{
+  "id": "eb645ef8-1373-41eb-bf67-6afcea7e2069",
+  "visible": true,
+  "type": "profile"
+}
+```
+
+Attributes:
+* id: the profile id that corresponds to the id of the eperson associated
+* visible: indicates whether the profile is public (visible: true) or private (otherwise)
+* type: the resource type (profile)
+
+Exposed links:
+* item: the item with which the profile is modeled
+* eperson: the eperson related to the profile
+
+It would respond with:
+* 200 OK - Returning the profile if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the profile
+* 404 Not found - if the profile doesn't exist
```

Comment:
I'm assuming this also would return a 404 if the EPerson doesn't exist (e.g. if it was deleted)